test/remote/recovery_key_tests.js

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Having remote tests is 👍, but please also add some local tests in `./test/local/routes/recovery-keys.js` for consistency with other routes.